tests/it/runtime/dependencies.d

```diff
@@ -224,10 +224,14 @@ unittest {
                 ]
             );
 
-            // but do write configuration if dub cfg has changed
-            auto lines = runIt("-d myvar=foo");
-            srcLine.should.not.be in lines;
-            cfgLine.should.be in lines;
+            // we cache dub's PackageManager per-thread; use a new thread to make sure the changed .sdl is reloaded
```

Comment:
This means the test needs to know waaay too much about the implementation.

Reply:
Well, the test needs to reparse a changed .sdl, something a real reggae process never does, and I thought this is the least invasive method to make that test still work.